### PR TITLE
Limit possible dimensions based on value count

### PR DESCRIPTION
**Pull request by kravets-levko**
For `time-series`: show only dimensions that have > 1 value; make visualizations available only if there is at least one dimensions with this condition.

For `sortable-series`: show only dimensions that have <= 10 values; make visualizations available only if there is at least one dimensions with this condition.

### CODE REVIEW

File: app/front/scripts/services/os-viewer/index.js [hunk 1]
```diff
@@ -54,7 +56,18 @@ function fullyPopulateModel(state) {
     .then(function(packageModel) {
       packageModel.columnHierarchies =
         getHierarchiesWithLimitedDimensionValues(packageModel.hierarchies,
-          maxDimensionValuesForColumns);
+          1, maxDimensionValuesForColumns);
+      // Get some special hierarchies
+      packageModel.specialHierarchies = {
+        sortableSeries: getHierarchiesWithLimitedDimensionValues(
+          packageModel.hierarchies, 1, 10),
+        timeSeries: getHierarchiesWithLimitedDimensionValues(
```
Comment (akariv) [inline app/front/scripts/services/os-viewer/index.js]: This should be just the datetime hierarchies.

File: app/front/scripts/directives/sidebar/template.html [hunk 2]
```diff
@@ -8,11 +8,11 @@
     params="params"></drilldown-sidebar>
 
   <sortable-series-sidebar ng-switch-when="sortable-series"
-    hierarchies="datapackage.hierarchies"
+    hierarchies="datapackage.specialHierarchies.sortableSeries"
```
Comment (akariv) [inline app/front/scripts/directives/sidebar/template.html]: This should only affect the series, not the rows.

**Review by akariv — CHANGES_REQUESTED** [review]

File: app/front/scripts/services/visualizations/index.js [hunk 3]
```diff
@@ -113,15 +113,15 @@ function getAvailableVisualizations(packageModel) {
...
+          (packageModel.dateTimeHierarchies.length > 0);
         if (!hierarchiesAvailable) {
           return false;
         }
       }
       if (item.type == 'sortable-series') {
-        hierarchiesAvailable = _.isObject(packageModel.specialHierarchies) &&
-          _.isArray(packageModel.specialHierarchies.sortableSeries) &&
-          (packageModel.specialHierarchies.sortableSeries.length > 0);
+        hierarchiesAvailable = _.isArray(packageModel.seriesHierarchies) &&
```
Comment (akariv) [inline app/front/scripts/services/visualizations/index.js]: It's actually OK to have just row hierarchies with no series.

**Review by akariv — CHANGES_REQUESTED** [review]

**Comment by kravets-levko** [comment]
@akariv Please review this PR again. Thanks!

**Pull request merged**